### PR TITLE
Add test that ensures that all the symbol layers are rendered

### PR DESCRIPTION
**Pull request by FrankApiyo**
Fixes: #251

### CODE REVIEW

**Comment by FrankApiyo** [comment]
Will add better description to this PR

File: src/configs/settings.ts [hunk 1]
```diff
@@ -205,6 +205,12 @@ export const PlanActivityTitles = [
   'BCC',
   'IRS',
 ] as const;
+
+export const goalsWithSymbols = [
```
Comment (moshthepitt) [inline src/configs/settings.ts]: You need to add the values from the `planActivities` object e.g. `Larval_Dipping` should also be added to this list

**Comment by FrankApiyo** [comment]
@moshthepitt do you think replacing a very large snapshot with smaller snapshots that will not necessarily usually fail at the same time is a good solution to reducing the sizes of snapshots. The snapshot file itself will still remain large. But individual snapshots will reduce in size significantly.

**Comment by moshthepitt** [comment]
> @moshthepitt do you think replacing a very large snapshot with smaller snapshots that will not necessarily usually fail at the same time is a good solution to reducing the sizes of snapshots. The snapshot file itself will still remain large. But individual snapshots will reduce in size significantly.
> 
> That is what I just did in this last commit

Yes I think this is better because it is more readable and requires less resources to validate that the spanshot is valid.

File: src/configs/settings.ts (hunk 1 above)
Comment (moshthepitt) [reply src/configs/settings.ts]: @FrankApiyo did you address this?

Comment (FrankApiyo) [reply src/configs/settings.ts]: Not yet. I'm on it.

Comment (moshthepitt) [reply src/configs/settings.ts]: Is there a reason why you have not added `Mosquito_Collection`?

**Comment by moshthepitt** [comment]
@FrankApiyo  what is the status of this?  Do you think you'll be able to get this closed, or should we close this PR?

**Comment by FrankApiyo** [comment]
> @FrankApiyo what is the status of this? Do you think you'll be able to get this closed, or should we close this PR?

@moshthepitt I don't think I'll be able to get this closed this week

**Comment by moshthepitt** [comment]
> @moshthepitt I don't think I'll be able to get this closed this week

@FrankApiyo could you document what you think the next steps for this are?

**Comment by FrankApiyo** [comment]
> > @moshthepitt I don't think I'll be able to get this closed this week
> 
> @FrankApiyo could you document what you think the next steps for this are?

@kahama94 has volunteered to take it from here

**Comment by moshthepitt** [comment]
@FrankApiyo that's fine but can you still document the next steps for this please?

**Comment by FrankApiyo** [comment]
> @FrankApiyo that's fine but can you still document the next steps for this please?

The following are the next steps for this PR IMHO:

- add the correct goals with symbols to the goals_with_symbols array in settings.ts

- attempt to reduce the sizes of the snapshots

**Comment by moshthepitt** [comment]
Looks like a stale PR and branch.

**Pull request closed without merging**